src/lib.rs

```diff
@@ -70,7 +71,7 @@ pub use crate::carrier::Carrier;
 mod phone_number;
 pub use crate::phone_number::{PhoneNumber, Type};
 
-mod parser;
+pub mod parser;
```

Comment:
I guess you made this public because of the benchmark.

But it's not supposed to be part of the public API so it probably should be `#[doc(hidden]` or similar.